### PR TITLE
feat: select

### PR DESCRIPTION
**Pull request by mfranzke**
## Proposed changes

Resolves https://github.com/db-ui/mono/issues/1365

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/angular-showcase/src/app/components/form/form.component.html
Comment (mfranzke) [inline showcases/angular-showcase/src/app/components/form/form.component.html:1]: this doesn't work so far, but we'll put this into a follow up story.

Comment (mfranzke) [reply showcases/angular-showcase/src/app/components/form/form.component.html:1]: https://github.com/db-ui/mono/issues/1537

**Comment by annsch** [comment]
@mfranzke it looks like the asterisk has a bit too much space:

Bildschirmfoto 2023-09-08 um 09 43 27

**Comment by mfranzke** [comment]
> @mfranzke it looks like the asterisk has a bit too much space:
> 
> Bildschirmfoto 2023-09-08 um 09 43 27

I'll cover this within https://github.com/db-ui/mono/pull/1545, as we should generalize it through all the different form elements.

**Pull request merged**